crates/relayer-framework/src/tests/relayer_mock/base/types/message.rs

```diff
@@ -0,0 +1,25 @@
+use std::fmt::{Display, Formatter, Result};
+
+use crate::tests::relayer_mock::base::types::{height::Height, packet::PacketKey};
+use crate::tests::relayer_mock::contexts::chain::{ChainState, ClientId};
+
+#[derive(Debug)]
+pub enum Message {
+    SendPacket(ClientId, Height, Height, PacketKey),
```

Comment:
This should be a `RecvPacket` message. SendPacket events are initiated by external transactions like IBC transfer.

We should also document what each of these messages correspond to in the actual IBC implementation.